yarn-install/action.yml

```diff
@@ -37,10 +40,17 @@ runs:
       id: cache
       with:
         path: '**/node_modules'
-        key: ${{ runner.os }}-node-${{ hashFiles('yarn.lock') }}-${{ inputs.dependencies }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}-${{ inputs.dependencies }}
+
+    - name: Set npm registry auth token
+      shell: bash
+      if: ${{ inputs.npm-auth-token != '' }}
```

Comment:
tää vois olla joko
`if: inputs.npm-auth-token != ''` koska GH evaluoi tuon anyways
tai 
`if: ${{ inputs.npm-auth-token }}` koska tyhjä stringi type castataan `0`:ksi kuten `false`kin 
https://docs.github.com/en/actions/learn-github-actions/expressions

Reply:
joo, totta. Mutta toimiiko `if: inputs.npm-auth-token`?

Reply:
pitäs kyl toimia, kun se evaluoidaan, ja tyhjä string == 0 joka on sama kun false. Näin mä ainakin ymmärrän dokumentaatiosta. 

Mut voihan sinne panna toisen testin, jossa kerrotaan että ko. tokenia ei ollut, tai logittaa sen pituutta tms.